.github/dependabot.yml

```diff
@@ -19,11 +19,3 @@ updates:
     directory: "/"
     schedule:
       interval: "monthly"
-
-  - package-ecosystem: "npm"
-    directory: "/ui"
-    schedule:
-      interval: "monthly"
-    ignore:
-      - dependency-name: "*"
-        update-types: ["version-update:semver-patch"]
```

Comment:
I can see how this one can get very annoying but I'd like to keep the UI up-to-date as much as possible (best effort here, if it doesn't pass don't merge it) so that tools like security warnings are not missed because of the multitude of issues reported. 

Alternatively if we're absolutely certain that we don't care about the ui/ I will remove it in a separate PR not to delay things anymore!